test/mocks/gen/pkg/services/file/file.go

```diff
@@ -0,0 +1,425 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
We need a clever way to remove old mock files when we make a rename like this. `test/mocks/gen/pkg/services/file.go` and `test/mocks/gen/pkg/services/program.go` should be removed... We need to think about something to improve the `Makefile` for that... Would you please remove these two file manually until we have a better `Makefile`?